app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarWeeklyViewModel.kt

```diff
@@ -79,6 +81,30 @@ class CalendarWeeklyViewModel @Inject constructor(
         return granted.containsAll(PERMISSIONS)
     }
 
+    suspend fun getDaysCaloriesActive(date: LocalDate): Double {
+        val end = getEndInstant(date)
+        val start = date
```

Comment:
Single empty line could be added, because of the multi-line block in start variable initialization.